### DISCUSSION TITLE
One-way roads.

### DISCUSSION THREAD

**Discussion by murolem** [voxeltycoon/issues#327] [General] [answered]
This will allow for better management of traffic

**Comment by randomizermk1** [comment]
a one way road with a speed multiplier was what I was envisioning
add a bit more darker texture, get rid of the side walks, make it the tech tree, make it curve like the rail ways

**Comment by victorleblais** [comment, marked as answer]
This feature is planned as part of the rework to the road and rail system, [described in the Early Access FAQ](https://voxeltycoon.xyz/devlog/ea-faq/).

There is already a post for this here: https://github.com/voxeltycoon/issues/discussions/340
